### PR TITLE
Feature/ukr 229 order type popup

### PR DESCRIPTION
**Pull request by lewis-jump24**
A small popup component to redirect the customer to the appropriate service page.

### CODE REVIEW

File: resources/js/components/shared/store/ServiceTypePopup.vue [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    <h2 class="body-xlarge margin-bottom-50">
+      Did you want to do a...
+    </h2>
+
+    <div class="content">
+      <div class="col-span-6">
+        <a
+          ref="take_away"
+          :href="takeAwayUrl"
+          class="button button--large button-solid--cloud-burst button--hover-casablanca button--focus-casablanca"
```
Comment (annekam) [inline resources/js/components/shared/store/ServiceTypePopup.vue:12]: should this be button-outline?

File: resources/sass/helpers/buttons/_button.scss [hunk 2]
```diff
@@ -29,9 +29,28 @@
   @extend .margin-right-10;
 }
 
+.button--large {
+  width: 100%;
```
Comment (annekam) [inline resources/sass/helpers/buttons/_button.scss:33]: i wouldn't add the 100% here as there maybe instances where we don't want the large button to take up the entire width of the container

File: resources/sass/helpers/buttons/_button.scss [hunk 3]
```diff
@@ -29,9 +29,28 @@
...
+  height: 100px;
+}
+
+.button--large .button__icon svg {
+  height: 35px;
```
Comment (annekam) [inline resources/sass/helpers/buttons/_button.scss:38]: this is too generic, not every svg for the button will need this height. This is something that is more component specific so maybe create a sub folder within popups called service-type and create a couple of custom styles there

File: resources/js/components/shared/store/ServiceTypePopup.vue [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+      type: String,
+      default: '',
+    },
+    tableServiceUrl: {
+      type: String,
+      default: '',
+    },
+  },
+  mounted() {
+    this.$refs.take_away.focus();
```
Comment (dannewnsjump24) [inline resources/js/components/shared/store/ServiceTypePopup.vue:58]: nice thought behind this. good work.

**Pull request merged**